### PR TITLE
blog: add august 8th pre-announcement security release

### PR DESCRIPTION
**Pull request by RafaelGSS**

### CODE REVIEW

**Review by bmuenzenmeyer — APPROVED** [review]

**Comment by ovflowd** [comment]
FYI: @RafaelGSS as you know, you don't need an ✅ from the Website team, you can simply just go ahead and merge once checks pass 🙇 (just a reminder if by chance you're waiting on us)

**Comment by bmuenzenmeyer** [comment]
> FYI: @RafaelGSS as you know, you don't need an ✅ from the Website team, you can simply just go ahead and merge once checks pass 🙇 (just a reminder if by chance you're waiting on us)

thanks for clarifying @ovflowd - also merged `main` now that CI is resolved

**Comment by RafaelGSS** [comment]
> FYI: @RafaelGSS as you know, you don't need an ✅ from the Website team, you can simply just go ahead and merge once checks pass 🙇 (just a reminder if by chance you're waiting on us)

I'm waiting for `Build Checks / Build on windows-latest (pull_request) In progress` which is required :)

**Comment by ovflowd** [comment]
> I'm waiting for Build Checks / Build on windows-latest (pull_request) In progress which is required :)

Yeah 100%, we've just applied some CI policies, so probably something got stuck before @bmuenzenmeyer rebase!

File: pages/en/blog/vulnerability/august-2023-security-releases.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+title: Tuesday August 8th 2023 Security Releases
+slug: august-2023-security-releases
+layout: blog-post.hbs
+author: Rafael Gonzaga
+---
+
+# Summary
+
+The Node.js project will release new versions of the 16.x, 18.x and 20.x
+releases lines on or shortly after, Tuesday August 1st 2023 in order to address:
```
Comment (addaleax) [inline pages/en/blog/vulnerability/august-2023-security-releases.md:13]: @RafaelGSS Should this have been `August 8th`?

Comment (RafaelGSS) [reply pages/en/blog/vulnerability/august-2023-security-releases.md:13]: Yes, fixed. Thx

**Pull request merged**